apps/app/src/components/SkipToTheContent.tsx

```diff
@@ -0,0 +1,8 @@
+export const SkipToTheContent = () => (
+	<a
```

Comment:
I need confirmation that I used proper tag - I'm not sure that we can't achieve that functionality by using the `Link` from `next/link`.

When I wanted to use `Link` component from `next/link` I realized that when I click on a link which contains for eg. `#element` at the end of the href attribute it does not cause scrolling page to the pointed element so I used native html `a` tag which works as expected.

Reply:
I found something about my concerns: https://github.com/vercel/next.js/issues/44295